src/main/java9/module-info.java

```diff
@@ -2,6 +2,7 @@
 
     requires transitive java.sql;
     requires transitive java.sql.rowset;
+    requires static org.graalvm.sdk;
```

Comment:
wouldn't that have any effects when running with a normal JDK?

Reply:
It shouldn't have.
It expresses a compile-time dependency that is optional at runtime
> There is also a `requires static` directive to indicate that a module is required at compile time, but is optional at runtime.

https://www.oracle.com/be/corporate/features/understanding-java-9-modules.html